policyengine_us/parameters/gov/states/co/tax/income/subtractions/charitable_contribution/amount.yaml

```diff
@@ -0,0 +1,17 @@
+description: Colorado allows filers to receive the deductions for charitable contributions with contributions above this amount.
```

Comment:
Deduction or subtraction?

Reply:
subtraction in law but deduction in form - used subtraction for